packages/plugins/integration-zapier-ui/src/lib/components/zapier-webhooks/zapier-webhooks.component.ts

```
@@ -0,0 +1,115 @@
+					if (!config.clientId || !config.clientSecret) {
+						throw new Error('Missing client credentials');
+					}
+					return this._zapierService.exchangeCodeForToken({
+						client_id: config.clientId,
+						client_secret: config.clientSecret,
+						redirect_uri: config.redirectUri,
+						grant_type: 'authorization_code'
+					});
+				}),
+				switchMap((tokens: IZapierAccessTokens) => {
+					if (tokens && tokens.access_token) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)